src/stactools/goes_glm/constants.py

```diff
@@ -175,3 +175,131 @@ class OrbitalSlot(str, enum.Enum):
         ]
     ],
 }
+
```

Comment:
The columns can't be constant as the columns vary slightly. Sometimes there are for example columns containing ...`frame_time_offset`..., sometimes they are missing (here it seems they are missing). So you need to read the files to get the actual content...

Reply:
Well, that's unfortunate! But that's for the note. I'll get that info from the parquet file / netcdf file then.

Reply:
Ignore my latest commit, which really mangles `parquet.create_asset`. I intend to clean it up.